.github/workflows/cve-weekly.yml

```diff
@@ -282,40 +282,40 @@ jobs:
           rm bin
       # </template: unlink_bin_step>
 
-  send-email-report:
-    name: Send weekly report
-    needs:
-      - test_cve_report_main
-      - test_cve_report_channels
-    if: success() || failure()
-    runs-on: [ self-hosted, regular ]
-    steps:
-      - name: Clear output directory
-        run: |
-          rm -rf out/
-      - name: Download reports artifact
-        uses: actions/download-artifact@v2
-        with:
-          name: cve-reports
-          path: out/
-      - name: Create report archive
-        working-directory: out/
-        run: |
-          zip -r cve-reports.zip ./*
-      - name: Send report
-        env:
-          MAIL_API_TOKEN: ${{ secrets.MAIL_API_TOKEN }}
-          MAIL_API_URL: ${{ secrets.MAIL_API_URL }}
-          MAIL_API_TRIVY_SENDER: ${{ secrets.MAIL_API_TRIVY_SENDER }}
-          MAIL_API_TRIVY_RECEIVER: ${{ secrets.MAIL_API_TRIVY_RECEIVER }}
-        run: |
-          curl -s --user "${MAIL_API_TOKEN}" \
-            "${MAIL_API_URL}" \
-            -F from="${MAIL_API_TRIVY_SENDER}" \
-            -F to="${MAIL_API_TRIVY_RECEIVER}" \
-            -F subject='Trivy reports' \
-            -F text='Hello! There are Weekly Trivy scan reports attached to this message.' \
-            -F attachment=@out/cve-reports.zip
-      - name: Clear output directory
-        run: |
-          rm -rf out/
+  # send-email-report:
```

Comment:
I guess that you comment it manually. It is wrong way.
https://github.com/deckhouse/deckhouse/actions/runs/6668585535/job/18126516667?pr=6352